validation/security-configs_test.go

```diff
@@ -45,7 +45,8 @@ var checkResolutions = func(t *testing.T, r standard.Resolutions, chainID uint64
 			got, err := getAddress(method, contractAddress, client)
 			require.NoErrorf(t, err, "problem calling %s.%s (%s)", contract, method, contractAddress)
 
-			assert.Equal(t, want, got, "%s.%s = %s, expected %s (%s)", contract, method, got, want, output)
+			// Use assert.True here for a concise output of failures, since failure info is sent to a slack channel
+			assert.True(t, want == got, "%s.%s = %s, expected %s (%s)", contract, method, got, want, output)
```

Comment:
Do you think we could roll this out to any other locations where we are comparing byte slices?

I wonder if it might actually be just as well to do 

```go
if want != got {
  t.Errorf("%s.%s = %s, expected %s (%s)", contract, method, got, want, output)
}
```
This will give us the most useful output only.

Reply:
Done, as discussed.